module/workflow/src/Infrastructure/Handler/Workflow/UpdateWorkflowCommandHandler.php

```diff
@@ -41,17 +42,16 @@ public function __invoke(UpdateWorkflowCommand $command)
 
         Assert::notNull($workflow);
 
-        foreach ($command->getStatuses() as $code => $status) {
-            if (!$workflow->hasStatus($code)) {
-                $workflow->addStatus($code, $status);
-            } else {
-                $workflow->changeStatus($code, $status);
+        foreach ($command->getStatuses() as $status) {
```

Comment:
What about order?

Reply:
Transitions give flow and  order of statuses